piws/views/primary.py

```diff
@@ -105,6 +97,14 @@ def _prepare_side_boxes(self, entity):
                      "data-toggle='tooltip' title='{2}'>&#8594;</a> "
                      "{1}".format(source_etype, target_etype, rschema.type))
 
+            # Construct rql depending on entity role
+            if role == "subject":
+                rql = "Any X WHERE X is {0}, E eid '{1}', E {2} X".format(target_etype,
```

Comment:
Quand CW optimise la requête vaut il mieux mettre 'X is {0}' à la fin de la requête?